specification/ai/ModelInference/models/common.tsp

```diff
@@ -0,0 +1,54 @@
+import "@typespec/rest";
```

Comment:
I can't fine `response_format` here (or any .tsp file). But I do see it in the GetChatCompletions_Maximum example.

Reply:
It's here: https://github.com/Azure/azure-rest-api-specs/raw/d579b4744aa86f1eb26d491ff2f7f3d5d753b150/specification/ai/ModelInference/models/chat_completions.tsp#L94